extensions/schema-compare/src/test/testUtils.ts

```diff
@@ -155,100 +155,105 @@ export function setDatabaseEndpointInfo(): mssql.SchemaCompareEndpointInfo {
-		ignoreAuthorizer: { value: false, description: sampleDesc, displayName: sampleName },
-		ignoreAnsiNulls: { value: false, description: sampleDesc, displayName: sampleName },
-		generateSmartDefaults: { value: false, description: sampleDesc, displayName: sampleName },
-		dropStatisticsNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-		dropRoleMembersNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-		dropPermissionsNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-		dropObjectsNotInSource: { value: false, description: sampleDesc, displayName: sampleName },
-		ignoreColumnOrder: { value: false, description: sampleDesc, displayName: sampleName },
-		doNotDropObjectTypes: { value: [], description: sampleDesc, displayName: sampleName },
-		excludeObjectTypes: { value: [], description: sampleDesc, displayName: sampleName },
-		ignoreTablePartitionOptions: { value: false, description: sampleDesc, displayName: sampleName },
-		doNotEvaluateSqlCmdVariables: { value: false, description: sampleDesc, displayName: sampleName },
-		disableParallelismForEnablingIndexes: { value: false, description: sampleDesc, displayName: sampleName },
-		disableIndexesForDataPhase: { value: false, description: sampleDesc, displayName: sampleName },
-		restoreSequenceCurrentValue: { value: false, description: sampleDesc, displayName: sampleName },
-		rebuildIndexesOfflineForDataPhase: { value: false, description: sampleDesc, displayName: sampleName },
-		isAlwaysEncryptedParameterizationEnabled: { value: false, description: sampleDesc, displayName: sampleName },
-		preserveIdentityLastValues: { value: false, description: sampleDesc, displayName: sampleName },
-		allowExternalLibraryPaths: { value: false, description: sampleDesc, displayName: sampleName },
-		allowExternalLanguagePaths: { value: false, description: sampleDesc, displayName: sampleName },
-		hashObjectNamesInLogs: { value: false, description: sampleDesc, displayName: sampleName },
-		doNotDropWorkloadClassifiers: { value: false, description: sampleDesc, displayName: sampleName },
-		ignoreWorkloadClassifiers: { value: false, description: sampleDesc, displayName: sampleName },
-		ignoreDatabaseWorkloadGroups: { value: false, description: sampleDesc, displayName: sampleName },
-		doNotDropDatabaseWorkloadGroups: { value: false, description: sampleDesc, displayName: sampleName }
+		ignoreTableOptions: { value: false, description: sampleDesc, propertyName: sampleName },
+		ignoreSemicolonBetweenStatements: { value: false, description: sampleDesc, propertyName: sampleName },
+		ignoreRouteLifetime: { value: false, description: sampleDesc, propertyName: sampleName },
+		ignoreRoleMembership: { value: false, description: sampleDesc, propertyName: sampleName },
+		ignoreQuotedIdentifiers: { value: false, description: sampleDesc, propertyName: sampleName },
```

Comment:
these hard coded options can be removed now that there's the optionsMapTable, right?

Reply:
Already removed! :)